### ISSUE TITLE
using   ts-patch-mongoose with typegoose

### ISSUE THREAD

**Issue by nourissam** [ilovepixelart/ts-patch-mongoose#24]
Can we use ts-patch-mongoose with typegosse class??

**Comment by ilovepixelart** [comment]
Yes you should be able to since this is a plugin for mongoose, if no consider creating a small project that I can start and check)
